devops/pipeline/stages/source_build.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# 
```

Comment:
Here and elsewhere.

The MIT license is stated at root of the project.
The notices at top of each source and infra file must be the  lone copyright notice.

Please remove the license line from this file and the rest.